future.go

```diff
@@ -0,0 +1,93 @@
+package binance
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+const (
+	futureAPIBaseURL = "https://fapi.binance.com"
+)
+
+// CreateFutureOrder ...
+func (bc *Client) CreateFutureOrder(symbol, side, positionSide, tradeType string, quantity float64) (CreateOrderResult, error) {
+	var (
+		response CreateOrderResult
```

Comment:
this response type is for spot, future API return more data https://binance-docs.github.io/apidocs/futures/en/#new-order-trade. the param tradeType is not used in this function and refer from the docs there are more params can be supported in this api